src/plugins/htmlEditor.ts

```diff
@@ -13,7 +15,7 @@ export default class HtmlEditor {
 
     enable() {
         window["CKEDITOR_BASEPATH"] = Config.CK_EDITOR_BASE_PATH;
-        $.getScript(Config.CK_EDITOR_BASE_PATH + "ckeditor.js", () => this.onCkEditorScriptReady());
+        this.onDemandScript(Config.CK_EDITOR_BASE_PATH + "ckeditor.js", () => this.onCkEditorScriptReady());
```

Comment:
This function is just like the `$.getScript()` but it caches the script file and we don't need to get the `ckeditor.js` file each time.